### ISSUE TITLE
Set up a deployment hook

### ISSUE THREAD

**Issue by ob-ivan** [ob-ivan/ommacomics#9]
After push to master: upload to production.

Benefits:
- Edit from the GitHub web interface. 
- Security PRs from dependabot are deployed as soon as merged.

**Comment by ob-ivan** [comment]
I have created `public/githook.php`.

Requesting http://comics.ob-ivan.ru/githook.php results in 404.

I've enabled rewrite log in the nginx config:

```
+    rewrite_log on;
+    error_log /var/log/nginx/comics.ob-ivan.ru.error.log debug;
```

An abriged excerpt from the log:

```
2023/02/13 04:28:54 [debug] 26171#26171: *1 http uri: "/githook.php"
...
2023/02/13 04:28:54 [debug] 26171#26171: *1 rewrite phase: 1
2023/02/13 04:28:54 [debug] 26171#26171: *1 test location: "/"
2023/02/13 04:28:54 [debug] 26171#26171: *1 test location: ~ "^/index\.php(/|$)"
2023/02/13 04:28:54 [debug] 26171#26171: *1 test location: ~ "\.php$"
2023/02/13 04:28:54 [debug] 26171#26171: *1 http finalize request: 404, "/githook.php?" a:1, c:1
```

Why didn't the location expression `"\.php$` match the request?

**Comment by ob-ivan** [comment]
The problem was that the location was marked as `internal` for whatever reason. I removed the `internal` command from the location, and it started to match.

**Comment by ob-ivan** [comment]
Error running `composer install` from the deploy script:

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  PHP Fatal error:  Uncaught RuntimeException: APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file. in /web/comics.ob-ivan.ru/bin/console:20
!!  Stack trace:
!!  #0 {main}
!!    thrown in /web/comics.ob-ivan.ru/bin/console on line 20
!!  
Script @auto-scripts was called via post-install-cmd
```

**Comment by ob-ivan** [comment]
Error running the hook:

```
/web/comics.ob-ivan.ru/public/../deploy.sh: line 32: /web/comics.ob-ivan.ru/deploy.lock: Permission denied
```

**Comment by ob-ivan** [comment]
Fixed the above error by moving the `deploy.lock` file to the `var` directory (62896b), but then this error occurs:

```
+ git pull
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

That's right, the web server user shall not have access to execute `git pull`. I need to come up with another approach.

**Comment by ob-ivan** [comment]
Suggestion:
* The hook places a file `var/deploy.request`.
* I create a cronjob that tries to read `var/deploy.request` every minute, and if it exists, starts `deploy.sh` and upon successful return removes `var/deploy.request`.

**Comment by ob-ivan** [comment]
Resolved via #74.
